packs/boons-and-curses/grandmother-spider-major-curse.json

```diff
@@ -4,7 +4,7 @@
     "name": "Grandmother Spider - Major Curse",
     "system": {
         "description": {
-            "value": "<p><strong>Major Curse:</strong> Grandmother Spider throws all of her might behind the underdog. Any time a creature with a level lower than yours rolls a success on a check against you, it critically succeeds instead; any time you roll a failure on a check against a creature with a level lower than yours, you critically fail instead.</p>"
+            "value": "<p>Grandmother Spider throws all of her might behind the underdog. Any time a creature with a level lower than yours rolls a success on a check against you, it critically succeeds instead; any time you roll a failure on a check against a creature with a level lower than yours, you critically fail instead.</p>"
```

Comment:
I think we can ephemeral the other part too. Fine to do as a follow up since I'd like to get this in for the release tonight